lighthouse-core/test/audits/unsized-images-test.js

```diff
@@ -234,6 +234,38 @@ describe('Sized images audit', () => {
       });
       expect(result.score).toEqual(1);
     });
+
+    it('passes when an image has attribute width/height of zero', async () => {
+      const result = await runAudit({
+        attributeWidth: '0',
+        attributeHeight: '0',
+        cssWidth: '',
+        cssHeight: '',
+        node: {
+          boundingRect: {
+            width: 0,
+            height: 0,
+          },
+        },
+      });
+      expect(result.score).toEqual(1);
+    });
+
+    it('passes when an image has is unsized, but its parent is not displayed..', async () => {
```

Comment:
```suggestion
    it('passes when an image has is unsized, but its parent is not displayed', async () => {
```

were those intentional?